cmd/api-linter/main.go

```diff
@@ -38,13 +38,10 @@ func runCLI(args []string) error {
 	return c.lint(globalRules, globalConfigs)
 }
 
+// Enable all rules by default.
+// Once we have rules other than core,
+// we will determine if we want to
```

Comment:
We 100% most certainly will, but this is still fine for now.

Reply:
Acknowledged.